source/_components/lightwave.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Lightwave"
+description: "Instructions on how to integrate Lightwave devices with Home Assistant."
+date: 2018-12-02 18:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lightwave.png
+ha_category: Hub
+ha_release: 0.1
+ha_iot_class: "Local Polling"
+---
+The `lightwave` component is the main component to set up and supported Lightwave lights and switches. Lights and switches need to be added in their own sections
```

Comment:
... to set up the supported Lightwave lights and switches.

Reply:
Missing period at the end.

Reply:
It's a bit unclear what we mean with "need to be added in their own sections".

Reply:
Revised the text.